packages/terra-status/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
At some point we need to add this (as well as NOTICE) to the existing projects (i.e. terra-button, etc.)  Should we just use this PR for that or create a new one?